<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QWidget" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_p1">
     <property name="title">
      <string>Player 1 ( Black )</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_name_p1">
        <property name="text">
         <string>&amp;Name :</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_name_p1</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name_p1"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>31</width>
          <height>41</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_type_p1">
        <property name="text">
         <string>&amp;Type :</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_type_p1</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_type_p1">
        <item>
         <property name="text">
          <string>Human</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer ( Basic )</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer ( Amateur )</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer ( Expert )</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="ok_pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_p2">
     <property name="title">
      <string>Player 2 ( White )</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_name_p2">
        <property name="text">
         <string>&amp;Name :</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_name_p2</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name_p2"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_type_p2">
        <property name="text">
         <string>&amp;Type :</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_type_p2</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_type_p2">
        <item>
         <property name="text">
          <string>Human</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer ( Basic )</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer ( Amateur )</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer ( Expert )</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name_p1</tabstop>
  <tabstop>comboBox_type_p1</tabstop>
  <tabstop>lineEdit_name_p2</tabstop>
  <tabstop>comboBox_type_p2</tabstop>
  <tabstop>ok_pushButton</tabstop>
  <tabstop>cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
